packages/xo-server/src/api/resource.js

```diff
@@ -0,0 +1,34 @@
+export async function downloadAndInstallResource({
```

Comment:
This method cannot work without the `xo-server-cloud` plugin and we download resources from the cloud. So it must be in the plugin code.

Reply:
I've followed what have been done with `XO-SAN`. Even `XO-SAN` uses `xo-server-cloud` and it has its own api in server-side.

Reply:
It's not the same things. `XO-SAN` is a product which use `xo-server-cloud`. Deploying a resource from the cloud is a cloud functionality.

Reply:
IMHO getting templates does depend on `xo-server-cloud`, because it requires a valid registration on our servers to be able to get the templates (cf our internal specification)